### DISCUSSION TITLE
How to get access to gitbook for modifying micro site content for new capabilities?

### DISCUSSION THREAD

**Discussion by aks30** [Sunbird-inQuiry/Community#10] [General Discussions]
How can we get access to the gitbook for modifying the microsite content with respect to new APIs and capabilities added?

cc - @joffinjoy @vijiurs @kiranharidas187

**Comment by pramodkvarma** [comment]
You can do PR on the markdown even without Gitbook. So, if you are not a regular core developer, maybe easier to simply do PR on the doc [Overview page](https://github.com/Sunbird-inQuiry/Community) and the rest of the docs as markdown files in folders [Lern](https://github.com/Sunbird-inQuiry/Community/tree/main/learn), [Use](https://github.com/Sunbird-inQuiry/Community/tree/main/use), and other folders there.

**Reply by aks30** [reply]
Sure this helps, will check it out! Thank you @pramodkvarma
